woocommerce/woocommerce-android
# issue/3651-product-price-update-bug-fix

Fixes #3651. This PR updates the product regular price and sale price to use `WCMaterialOutlinedEditTextView` instead of using `CurrencyEditText`. 

### Changes
- Added the currency symbol and currency position as part of the `SiteParameters`.
- Updates `ProductDetailViewModel` to fetch the store currency and displays that as a prefix or suffix of the currency value, based on the store settings.
- Removed rounding up of the sale and regular price.

### Screenshots
<img src="https://user-images.githubusercontent.com/22608780/116379104-53a2b200-a830-11eb-829a-e07491ea3500.gif" width="300"/>

### Testing
- Click on a product from the Products TAB.
- Click on the Pricing section.
- Notice that the currency symbol is displayed correctly and the position of the currency is displayed correctly.
- Try updating the regular price and the sale price and verify that the update is successful.

**This PR cannot be merged till this [FluxC PR](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1975) can be merged.**

Update release notes:
